jmx/jmx-bundle/pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>org.apache.aries.jmx</groupId>
         <artifactId>org.apache.aries.jmx.parent</artifactId>
-        <version>1.1.6-SNAPSHOT</version>
+        <version>1.1.5</version>
```

Comment:
Out of curiosity, why did you switch back to a release version for this module?

Reply:
The parent artifact has its parent updated (by you in #295 ) and I tested it during creation of workflows.
My goal is to prepare green workflows before making any changes, so the solution may not be perfect but... it works :) 
Without changing the version here to the last published version there are test errors in itest:
```
java.lang.ClassNotFoundException: org.osgi.jmx.framework.*
```
I wrote about the root reason in https://github.com/apache/aries/pull/296#discussion_r1863781179

Reply:
Ah, I missed that this is about the parent pom, sorry. So no concerns here.

Final comment from me - would it make sense to build the whole reactor instead of all the ITs and dependant projects?

Reply:
I am not able to force jmx/jmx-whiteboard to pass tests in reactor from zero:
1. current code has problems to fetch artifacts because of http addresses
2. With upgraded parent, pax url and added javax.inject the code compiles but tests failes and btw artifacts are downloaded via https so...
3. when removing upgrades and running build again the tests passes

I think that many other tests needs changes for new parent :( 

For your @rombert  tests it should be fine to add new tests to itest and `-am` option should include your new bundle build and test.

Reply:
Sounds good, thank you.